cyipopt/tests/unit/test_scipy_ipopt_from_scipy.py

```diff
@@ -465,29 +332,21 @@ def f(x):
         cons_ul = [{'type': 'ineq', 'fun': lambda x: 0 - x},
                    {'type': 'ineq', 'fun': lambda x: x + 1}]
 
-        sol = minimize(f, [10], method='slsqp', constraints=cons_u)
-        assert_(sol.success)
-        assert_allclose(sol.x, 0, atol=1e-10)
-
-        sol = minimize(f, [-10], method='slsqp', constraints=cons_l)
-        assert_(sol.success)
-        assert_allclose(sol.x, 2, atol=1e-10)
```

Comment:
The changes below look substantial, but really I only changed `method` to `None` and removed the last two tests because Ipopt thinks they are infeasible.